### ISSUE TITLE
BUG - Unknown Pokestop - loop

### ISSUE THREAD

**Issue by gpires** [PokemonGoF/PokemonGo-Bot#694]
### Actual Behavior

```
[2016-07-24 17:21:23] [#] Now at Pokestop: Unknown - Spinning...
[2016-07-24 17:21:31] [#] Found fort 59c06bde6fd74842a12f99543798844c.16 at distance 0.78km
[2016-07-24 17:21:31] [#] Need to move closer to Pokestop
[2016-07-24 17:21:31] [#] Walking from (38.716111, -9.136388) to (38.715852, -9.135778) for approx. 20.00 seconds
[2016-07-24 17:21:58] [#] Finished walking
[2016-07-24 17:21:58] [#] Arrived at Pokestop
[2016-07-24 17:22:00] [#] Now at Pokestop: Unknown - Spinning...
[2016-07-24 17:22:11] [#] Found fort ccc04c1054704662a76b134644d9262a.16 at distance 0.80km
[2016-07-24 17:22:11] [#] Need to move closer to Pokestop
[2016-07-24 17:22:11] [#] Walking from (38.715852, -9.135778) to (38.715271, -9.136994) for approx. 40.00 seconds
[2016-07-24 17:23:15] [#] Finished walking
[2016-07-24 17:23:15] [#] Arrived at Pokestop
[2016-07-24 17:23:18] [#] Now at Pokestop: Unknown - Spinning...
[2016-07-24 17:23:31] [#] Found fort 886f4d8eaeaf44c69a2b0f02983bf287.16 at distance 0.86km
[2016-07-24 17:23:31] [#] Need to move closer to Pokestop
[2016-07-24 17:23:31] [#] Walking from (38.715271, -9.136994) to (38.715325, -9.134854) for approx. 59.00 seconds
[2016-07-24 17:24:53] [#] Finished walking
[2016-07-24 17:24:53] [#] Arrived at Pokestop
[2016-07-24 17:24:56] [#] Now at Pokestop: Unknown - Spinning...
[2016-07-24 17:25:07] [#] Found fort db0848fe9e804cbf9e91356f7c98032b.16 at distance 0.91km
[2016-07-24 17:25:07] [#] Need to move closer to Pokestop
[2016-07-24 17:25:07] [#] Walking from (38.715325, -9.134854) to (38.714655, -9.135349) for approx. 28.00 seconds
[2016-07-24 17:25:51] [#] Finished walking
```
### Steps to Reproduce
Running in Lisbon :D

### Other Information
OS: Win10 
Git Commit: Last Dev

**Comment by MikeDX** [comment]
Duplicate of #643 and #676

**Comment by surfaace** [comment]
duplicate of #643, maybe fixed in #701
